api/search.go

```diff
@@ -13,6 +13,9 @@ import (
 func configGinSearchRouter(router *gin.Engine, daoWrapper dao.DaoWrapper) {
 	routes := searchRoutes{daoWrapper}
 	router.GET("/api/search/streams", routes.searchStreams)
+	g := router.Group("/api/search/subtitles")
```

Comment:
Minor:
We could theoretically group it as `/api/search` and then `.GET("/streams")` 
and `.GET("subtitles")`. Makes the code cleaner, I guess.

Reply:
Done :ok: Tho I was thinking about removing the ocr search considering the quality of the subtitles, this is something we can discuss on Wednesday :)